tests/requirements.txt

```diff
@@ -1,6 +1,22 @@
-matplotlib
-mypy==0.950
+# Tools used for testing
+docopt
+mypy==1.11.*
 pyright
+
+# Typed libraries and stubs
+matplotlib>=3.8
 pytest
+scipy-stubs
+
+# Untyped libraries, used to prevent "reportMissingImports" and get inferred typing
+joblib
+networkx
+PyOpenGL
+scikit-image
 scikit-learn
+transformers
+vispy
+numpydoc
+pyamg
```

Comment:
`numpydoc` and `pyamg` look like they may be unintentional re-exports and may possibly be removed from here if the export is removed from the stubs.

Reply:
Done in https://github.com/microsoft/python-type-stubs/pull/327